src/client/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -29,6 +29,30 @@ internal PublicClientApplication(ApplicationConfiguration configuration)
         {
         }
 
+        private const string CurrentBrokerAccountDescriptor = "current_broker_account";
+        private static IAccount s_currentBrokerAccount = 
+            new Account(CurrentBrokerAccountDescriptor, null, null, null);
+
+        /// <summary>
+        /// A special account value that indicates that the current Operating System account should be used 
+        /// to login the user. Not all operating systems and authentication flows support this concept, in which 
+        /// case calling `AcquireTokenSilent` will throw an <see cref="MsalUiRequiredException"/>. 
+        /// 
+        /// Currently only the Windows broker is able to login the current Windows user, see https://aka.ms/msal-net-wam for details.
+        /// </summary>
+        public static IAccount CurrentBrokerAccount
```

Comment:
@jmprieur  - how does this sound? Happy to take renaming suggestings. Currently this only works for WAM, and there's no plan to use "current account" concept in other brokers. Perhaps `CurrentWAMAccount` is better?

"Current account" concept also potentially makes sense in IWA (not covered here) and FLW scenarios (which we don't support)

Reply:
Also please review the public comments.

Reply:
I think CurrentBrokerAccount is more flexible, as you mention there could be other use cases outside of WAM. My suggestion would be to move the comment on line 41 to the very beginning (like 37). A developer may not realize it's no available on iOS and not read the entire comment, so starting w/the limitation might be a good idea. also, is this the default user?

---
In reply to: [525507602](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2197#discussion_r525507602) [](ancestors = 525507602)

Reply:
Also, i think the wiki page you made is really very well done. thanks.

---
In reply to: [525587965](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2197#discussion_r525587965) [](ancestors = 525587965,525507602)

Reply:
From an experience point of view this is the account currently logged in in the OS session. But I guess CurrentBrokerAccount could work. I'm not sure people know necessarily what a broker is, and that's an implementation detail?. What about OperatingSystemCurrentAccount? Or OperatingSystemSessionAccount? or OperatingSystemAccount?

Reply:
We can do `OperatingSystemCurrentAccount` but note that in the absence of WAM, MSAL will throw a UiRequiredException. Let's see if people get confused by this or not. At some point we may want to say AcquireTokenSilent(OperatingSystemCurrentAccount) = IWA but I'm rather cautious of that since IWA is such a pain to setup.

Reply:
What do you mean, @bgavrilMS : it's a pain to setup?